packages/services/f-wdio-utils/src/page.object.js

```diff
@@ -19,8 +21,8 @@ class Page {
         return this;
     }
 
-    waitForComponent (component) {
-        component.waitForExist();
+    waitForComponent (component, timeoutMs = this.#defaultWaitTimeout) {
```

Comment:
Is there any benefit of setting this on a private field rather than inline?

Reply:
x2 reason really, one so I can give the value a name and secondly so I can get rid of the annoying eslint warning :)